source/2019-12-30-countdown-to-the-new-year-2019-ember-exam.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Countdown to The New Year: ember-exam
+author: Jessica Jordan
+tags: 2019, Addons, DecEmber, Testing
+alias: 2019/12/30/countdown-to-the-new-year-2019-ember-exam
+responsive: true
+---
+
+This is already the second to last edition of our December series - "Countdown to The New Year: 31 Days of Ember Addons". In this series we are highlighting a new addon each day until the new year, and we hope you'll join us for the fun!
+
+## Day 30
+
+On the 30th day of our **Countdown to The New Year** we want to feature an exciting Ember addon to make your **test suite** fun to work with: **ember-exam**!
+
+Learn more about [ember-exam on Ember Observer](https://emberobserver.com/addons/ember-exam).
+
+### What It Does
+
+ember-exam is an addon that allows to have more control over running your ember-qunit (Ember's default testing framework) and ember-mocha test suite. It extends from the default Ember CLI `test` command, allowing you run it as you would `ember test`.
+
+### Why I Like It
+
+ember-exam allows us to run tests in parallel which can greatly reduce the overall run time of our test suite. Using the `--random` option of this addon, we are able to run tests in random order; this helps us to uncover interdependencies between test cases which would be hard to spot otherwise. And the `exam:iterate` allows us to iterate over parts of our test suite repeatedly when tracking down brittle tests.
```

Comment:
```suggestion
ember-exam allows us to run tests in parallel which can greatly reduce the overall run time of our test suite. Using the `--random` option of this addon, we are able to run tests in random order; this helps us to uncover interdependencies between test cases which would be hard to spot otherwise. And the `exam:iterate` option allows us to iterate over parts of our test suite repeatedly when tracking down brittle tests.
```